### ISSUE TITLE
The documentation could use a warning about the difference between Loops and Rec.Loops

### ISSUE THREAD

**Issue by Ptival** [mlang/purescript-monad-loops#5]
Just got bit by some stack overflows because I was using the non-`Rec` one in `Eff`.

**Comment by jamesdbrock** [comment]
Yeah, maybe the non-`Rec` module should be deprecated? Why would anyone ever want to use the non-tail-call, stack-exploding loops?
